cache/policy.go

```diff
@@ -248,7 +260,7 @@ func (p *sampledLFU) del(key uint64) {
 }
 
 func (p *sampledLFU) add(key uint64, cost int64) {
-	p.keyCosts[key] = cost
+	p.keyCosts[key] += cost
```

Comment:
What if the same key being added multiple times?

Reply:
Seems `add` is never called when key already exists.

Reply:
when key already exists, cost would be the cost change of value.